omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -157,158 +189,66 @@ able to build the server, i.e.
 
 .. include:: development-server
 
-Additional OMERO requirements
------------------------------
-
-PostgreSQL
-^^^^^^^^^^
-
-Install PostgreSQL::
-
-    $ brew install postgresql
-    $ postgres --version
-    postgres (PostgreSQL) 9.4.1
-
-.. _python_dependencies:
-
-Python dependencies
-^^^^^^^^^^^^^^^^^^^
-
-The Python dependencies can be installed in the system-wide Python
-:file:`site-packages`, in the Homebrew Python :file:`site-packages` or
-within a virtual environment. If you are using the system-wide Python
-:file:`site-packages`, you may need to use :program:`sudo` to install
-the dependencies. If you are using a virtual environment, activate it
-before calling the Python dependencies installation script.
-
-If you installed OMERO using Homebrew, execute the ``omero_python_deps``
-script::
-
-    $ omero_python_deps
-
-If you use a development server, execute the ``python_deps.sh`` script under
-:file:`docs/install`::
+Then change into the `dist` directory::
 
-    $ cd ~/code/projects/OMERO
-    $ docs/install/python_deps.sh
-
-If you encounter problems with the installation script, please take a look at
-:ref:`install_homebrew_common_issues`.
-
-Configuration
--------------
-
-Database creation
-^^^^^^^^^^^^^^^^^
-
-#. Create a new database cluster::
-
-    $ initdb -E UTF8 /usr/local/var/postgres
-
-#. Start the PostgreSQL server.
-
-   If you wish to use ``brew services`` to start postgresql, you will
-   first need to install it, then start the service::
-
-    $ brew tap gapple/services
-    $ brew services start postgresql
-
-   Alternatively, to start by hand::
-
-    $ pg_ctl -D /usr/local/var/postgres/ -l /usr/local/var/postgres/server.log start
-
-   It is also possible to configure :program:`launchd` to start
-   PostgreSQL at boot
-
-#. Create a user and database::
-
-    $ createuser -P -D -R -S db_user
-    Enter password for new role:       # enter db_password
-    Enter it again:                    # enter db_password
-    $ createdb -E UTF8 -O db_user omero_database
+    $ cd ~/code/projects/OMERO/dist
 
-   Note ``db_user`` and ``db_password`` should be replaced with a
-   username and password of your choice.  Record these for future use.
+and follow the steps for setting up the database and OMERO data directory like mentioned
+in the previous section.
```

Comment:
That's included via the ```.. include:: development-server```. It will be clearer when you see it on the staging page. Check again with tomorrow's build.